fpsim/locations/nuhdss.py

```diff
@@ -0,0 +1,1252 @@
+    mortality['f_spline'] = f_mortality_spline
+
+    return mortality
+
+
+def maternal_mortality():
+    '''
+    From World Bank indicators for maternal mortality ratio (modeled estimate) per 100,000 live births:
+    https://data.worldbank.org/indicator/SH.STA.MMRT?locations=KE
+    '''
+
+    data = np.array([
+        [2000, 708],
+        [2001, 702],
+        [2002, 692],
+        [2003, 678],
+        [2004, 653],
+        [2005, 618],
+        [2006, 583],
+        [2007, 545],
+        [2008, 513],
+        [2009, 472],
+        [2010, 432],
+        [2011, 398],
+        [2012, 373],
+        [2013, 364],
+        [2014, 358],
+        [2015, 353],
+        [2016, 346],
+        [2017, 342],
+
+    ])
+
+    maternal_mortality = {}
+    maternal_mortality['year'] = data[:, 0]
+    maternal_mortality['probs'] = data[:, 1] / 100000  # ratio per 100,000 live births
+    # maternal_mortality['ages'] = np.array([16, 17,   19, 22,   25, 50])
+    # maternal_mortality['age_probs'] = np.array([2.28, 1.63, 1.3, 1.12, 1.0, 1.0]) #need to be added
+
+    return maternal_mortality
+
+
+def infant_mortality():
+    '''
+    From World Bank indicators for infant mortality (< 1 year) for Kenya, per 1000 live births
+    From API_SP.DYN.IMRT.IN_DS2_en_excel_v2_1495452.numbers
+    Adolescent increased risk of infant mortality gradient taken
+    from Noori et al for Sub-Saharan African from 2014-2018.  Odds ratios with age 23-25 as reference group:
+    https://www.medrxiv.org/content/10.1101/2021.06.10.21258227v1
+    '''
+
+    data = np.array([
+        [1960, 118.1],
+        [1961, 113.7],
+        [1962, 109.8],
+        [1963, 106.5],
+        [1964, 103.8],
+        [1965, 101.6],
+        [1966, 99.8],
+        [1967, 98.0],
+        [1968, 96.3],
+        [1969, 94.5],
+        [1970, 92.6],
+        [1971, 90.8],
+        [1972, 88.8],
+        [1973, 86.7],
+        [1974, 84.5],
+        [1975, 82.2],
+        [1976, 79.8],
+        [1977, 77.4],
+        [1978, 75.0],
+        [1979, 72.7],
+        [1980, 70.5],
+        [1981, 68.5],
+        [1982, 66.7],
+        [1983, 65.1],
+        [1984, 63.8],
+        [1985, 62.9],
+        [1986, 62.4],
+        [1987, 62.4],
+        [1988, 62.8],
+        [1989, 63.7],
+        [1990, 64.8],
+        [1991, 66.1],
+        [1992, 67.2],
+        [1993, 67.9],
+        [1994, 68.0],
+        [1995, 67.6],
+        [1996, 66.5],
+        [1997, 65.1],
+        [1998, 63.5],
+        [1999, 61.7],
+        [2000, 59.7],
+        [2001, 57.6],
+        [2002, 55.4],
+        [2003, 53.1],
+        [2004, 50.7],
+        [2005, 48.1],
+        [2006, 45.8],
+        [2007, 43.8],
+        [2008, 41.4],
+        [2009, 40.3],
+        [2010, 39.4],
+        [2011, 38.6],
+        [2012, 38.2],
+        [2013, 37.5],
+        [2014, 36.5],
+        [2015, 35.3],
+        [2016, 34.5],
+        [2017, 33.9],
+        [2018, 32.8],
+        [2019, 31.9]
+    ])
```

Comment:
Is maternal mortality available in the NUHDSS data? If so, might be worth seeing if you can use it for the later dates.